### PR TITLE
refresh feh.profile

### PR DESCRIPTION
**Pull request by pirate486743186**

### CODE REVIEW

File: etc/profile-a-l/feh.profile [hunk 1]
```diff
@@ -7,30 +7,45 @@ include feh.local
...
 include globals.local
 
+noblacklist ${HOME}/.config/feh
+
 include disable-common.inc
 include disable-devel.inc
 include disable-exec.inc
 include disable-interpreters.inc
 include disable-programs.inc
 include disable-shell.inc
```
Comment (glitsj16) [inline etc/profile-a-l/feh.profile:18]: I think we can harden this further and make it a full whitelisting profile:

```
mkdir ${HOME}/.config/feh
whitelist ${HOME}/.config/feh
whitelist ${DOWNLOADS}
whitelist ${PICTURES}
include whitelist-common.inc
```

**Review by glitsj16 — COMMENTED** [review]
Nice work. We might harden this more (see comments)

**Review by glitsj16 — APPROVED** [review]
LGTM

File: etc/profile-a-l/feh.profile [hunk 2]
```diff
@@ -7,30 +7,46 @@ include feh.local
...
+include disable-proc.inc
 include disable-programs.inc
 include disable-shell.inc
 
+whitelist /usr/share/feh
+whitelist /usr/share/yudit
+include whitelist-run-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
```
Comment (kmk3) [inline etc/profile-a-l/feh.profile:25]:
```suggestion
include whitelist-run-common.inc
include whitelist-runuser-common.inc
```

There can be image files in /usr/share (such as in /usr/share/icon and
/usr/share/doc) and maybe in /var/games, so I'd avoid whitelisting them.

**Review by kmk3 — APPROVED** [review]

**Comment by netblue30** [comment]
merging, thanks!

**Pull request merged**